app/Policies/BookingPolicy.php

```diff
@@ -29,7 +31,9 @@ public function view()
      */
     public function create(User $user)
     {
-        return $user->rating >= 3 || $user->getActiveTraining(1) != null || $user->isModeratorOrAbove();
+        return ($user->active && $user->rating >= VatsimRating::S1->value
```

Comment:
This allows all S1's to see the booking "widget" on the index page, but that's not necessary if they don't have an active training or endorsement to book a position.

Tested with a user with no endorsements, S1 rating and no active trainings. Saw this widget, could write in all information, but then 403'ed on submit. Better just to hide this all together. Except for those who already are in training or have a infinite S1 endorsement

Reply:
Good catch. Going for a simple approach with two different checks.